schemas/index.ts

```diff
@@ -29,6 +29,7 @@ export const keySchema = datedSchema.extend({
 
 export const supportedCurrencySchema = z.enum([
   'CAD',
+  'CHF',
```

Comment:
For reference, this was the only change I actually made, everything else was generated afterwards